src/content/docs/logs/log-management/log-api/introduction-log-api.mdx

```diff
@@ -674,15 +700,11 @@ Will be treated as:
 }
 ```
 
-<Callout variant="important">
-  Log management does not support white space in attribute names. For example, `{"Sample Attribute": "Value"}` would cause errors.
-</Callout>
+## Log JSON examples [#log-attribute-examples]
 
-## Log JSON example [#log-attribute-examples]
+Attributes can be scalar JSON types like string and number. They can also be compound (or [nested](https://www.digitalocean.com/community/tutorials/an-introduction-to-json#working-with-complex-types-in-json)) objects. Compound attributes will have their leaf attributes stored with flattened names.
 
```

Comment:
Tense confusion here -- "will have" vs "have". Also, not sure "leaf attribute" is standard usage, maybe "their associated attributes" or something more literal like that?